Schema/PublicSchema/AvailabilityData.bond

```diff
@@ -0,0 +1,49 @@
+import "Domain.bond"
+import "TestResult.bond"
+
+namespace AI
+
+struct AvailabilityData
+    : Domain
+{
+    [Description("Schema version")]
+    10: required int32 	 ver = 2;
+    
+    [MaxStringLength("64")]
+    [Question("This is a new limit. Verify it's ok")]
+    20: required string 	 testRunId;
```

Comment:
This is availability documents that we didn't cleaned up yet. It will be done in the next iteration